code/modules/mob/living/brain/brain_item.dm

```diff
@@ -113,24 +113,34 @@
 	if(istype(O, /obj/item/borg/apparatus/organ_storage))
 		return //Borg organ bags shouldn't be killing brains
 
-	if((organ_flags & ORGAN_FAILING) && O.is_drainable() && O.reagents.has_reagent(/datum/reagent/medicine/mannitol)) //attempt to heal the brain
+	var/healby = 2 //heals 2 damage per unit of mannitol
+	var/pour = "carefully pour"
+	var/onto = "onto"
+	var/valid_container = TRUE
+	if(istype(O, /obj/item/reagent_containers/syringe))
+		healby = 4 //heal 4 damage because PRECISCION
+		pour = "precisely inject"
+		onto = "into"
+	else if(istype(O, /obj/item/reagent_containers/dropper))
+		healby = 4
```

Comment:
is getting 100% purity really worth 4x healing im really afraid fermichem is going to end up buffing all these items and then we find out someone already has a baby dumb way of purifying these chems.

Reply:
why 4x? default 0.75 purity mannitol heals 2 damage per unit. with syringe/dropper it's 4 per unit, so 2x buff. and 100% purity heals 2.6 damage per unit.

Reply:
mental